build.gradle

```
@@ -16,4 +16,3 @@ plugins {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
```

```suggestion
}

```
I would suggest re-adding this blank line back to solve your issues with Cirrus CI.

---

Thanks! I had to to it for both build.gradle files, what a sneaky bug!